packages/components/notifications/index.js

```diff
@@ -1,4 +1,6 @@
-/* eslint-disable import/prefer-default-export */
-export { default as ContentNotification } from '../../../src/components/notifications/content-notification';
-
-export { default as version } from './version';
+// This file exists  because we want jest to use our non-compiled code to run tests
+// if this file is missing, and you have a `module` or `main` that points to a non-existing file
+// (ie, a bundle that hasn't been built yet) then jest will fail if the bundle is not yet built.
+// all apps should export all their named exports from their root index.js
+export { default as ContentNotification } from './content-notification';
+export * from './content-notification';
```

Comment:
Left this `index.js` file so that the component can be imported from its own package